docs/sources/tempo/operations/grafana_datasource.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Grafana Tempo Datasource
+description: Use the Tempo Operator to deploy Tempo and use it as a Datasource with Grafana
+aliases:
+ - /docs/tempo/latest/gateway
+ - /docs/tempo/operations/gateway
+weight: 15
+---
+
+# Grafana Tempo Datasource
```

Comment:
The Grafana UI does not capitalize data source as as proper noun, so I've changed all occurances of data source to lower case. 
![image](https://github.com/grafana/tempo/assets/104772500/03ba5610-e60c-4525-9546-9a5bd3e19544)


```suggestion
# Grafana Tempo data source
```